workflow/executor/resource.go

```diff
@@ -0,0 +1,188 @@
+package executor
```

Comment:
Has any of the code below changed or has it simply been moved to a different file?

Reply:
It was just moved.